packages/nc-gui/components/account/Token.vue

```diff
@@ -108,123 +106,59 @@ const descriptionInput: VNodeRef = (el) => (el as HTMLInputElement)?.focus()
-          <template #default="{ text, record }">
-            <div class="w-[320px]">
-              <span v-if="record.show">{{ text }}</span>
-              <span v-else>*******************************************</span>
-            </div>
-          </template>
-        </a-table-column>
-
-        <!-- Actions -->
-
-        <a-table-column key="actions" :title="$t('labels.actions')" data-index="token">
-          <template #default="{ record }">
-            <div class="flex items-center gap-2">
-              <a-tooltip placement="bottom">
-                <template #title>
-                  <span v-if="record.show"> {{ $t('general.hide') }} </span>
-                  <span v-else> {{ $t('general.show') }} </span>
-                </template>
-
-                <a-button type="text" class="!rounded-md nc-toggle-token-visibility" @click="record.show = !record.show">
-                  <template #icon>
-                    <MaterialSymbolsVisibilityOff v-if="record.show" class="flex mx-auto h-[1.1rem]" />
-                    <MaterialSymbolsVisibility v-else class="flex mx-auto h-[1rem]" />
-                  </template>
-                </a-button>
-              </a-tooltip>
-
-              <a-tooltip placement="bottom">
-                <template #title> {{ $t('general.copy') }}</template>
-
-                <a-button type="text" class="!rounded-md" @click="copyToken(record.token)">
-                  <template #icon>
-                    <component :is="iconMap.copy" class="flex mx-auto h-[1rem]" />
-                  </template>
-                </a-button>
-              </a-tooltip>
-
-              <a-dropdown
-                :trigger="['click']"
-                class="flex"
-                placement="bottomRight"
-                overlay-class-name="nc-dropdown-api-token-mgmt"
-              >
-                <div class="flex flex-row items-center">
-                  <a-button type="text" class="!px-0">
-                    <div class="flex flex-row items-center h-[1.2rem]">
-                      <IcBaselineMoreVert class="nc-token-menu" />
-                    </div>
-                  </a-button>
-                </div>
-
-                <template #overlay>
-                  <a-menu data-testid="nc-token-row-action-icon">
-                    <a-menu-item>
-                      <div
-                        class="flex flex-row items-center py-3 h-[2rem] nc-delete-token"
-                        @click="openModal(record.token, record.description)"
-                      >
-                        <component :is="iconMap.delete" class="flex" />
-                        <div class="text-sm pl-2">{{ $t('general.remove') }}</div>
-                      </div>
-                    </a-menu-item>
-                  </a-menu>
-                </template>
-              </a-dropdown>
+      <span>Create personal API tokens to use in automation or external apps.</span>
+      <div class="w-full mt-5 border-1 rounded-md">
```

Comment:
what is the reason of ditching `<a-table>`?

Reply:
![image](https://github.com/nocodb/nocodb/assets/66205494/60d8e60a-ee39-413e-8d7e-776683ba5628)


we want to insert  this style inside the table  but I looked a lot to insert custom element inside ant table and didn't wor as expected